thoth/sourcemanagement/__init__.py

```diff
@@ -15,4 +15,4 @@
 # You should have received a copy of the GNU General Public License
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 
-__version__ = "0.2.6"
+__version__ = "0.2.7"
```

Comment:
The version should not be bumped, please check it :+1:

Reply:
I will revert it back, but I pushed it to pypi, I guess that was a mistake. Any suggestions?
https://pypi.org/project/thoth-sourcemanagement/

Reply:
OK, so let it as is in that case.